client/browser/scripts/test-integration.js

```diff
@@ -5,7 +5,7 @@ const { decompressRecordings, deleteRecordings } = require('./utils')
 // eslint-disable-next-line no-void
 void (async () => {
   await decompressRecordings()
-  shelljs.exec('POLLYJS_MODE=replay yarn run -T percy exec --quiet -- yarn run-integration', async code => {
+  shelljs.exec('POLLYJS_MODE=replay percy exec --quiet -- pnpm run-integration', async code => {
```

Comment:
The call to `percy` doesn't need `pnpm`?

Reply:
I'll try to check that one. I think if this script was run using pnpm then `percy` will be on the PATH though. Unlike `run-integration` which is a package.json script and not in `node_modules/.bin`...